content/en/docs/concepts/basics/overview.md

```diff
@@ -14,9 +14,9 @@ weight: 0
 
 There has been a huge demand from the developer community for Filecoin compatibility with [Ethereum](https://ethereum.org/en/what-is-ethereum/) and [Solidity](https://en.wikipedia.org/wiki/Solidity) out of the box. With the FVM, developers can access the massive corpus of audited and battle-tested smart contracts written in Solidity for the Ethereum Virtual Machine (EVM).
 
-The FVM also supports other foreign runtimes and virtual machines, anything that compiles to [WebAssembly (WASM)](https://developer.mozilla.org/en-US/docs/WebAssembly). WASM was built for modern web browsers and supports multiple languages, including Rust and Go, which provides the flexibility we need to meet a variety of needs.
+The FVM also supports other foreign runtimes and virtual machines, anything that compiles to [WebAssembly (WASM)](https://developer.mozilla.org/en-US/docs/WebAssembly). WASM provides the flexibility we need to meet a variety of needs, because it was built for modern web browsers and supports multiple languages, including Rust and Go.
```

Comment:
```suggestion
The FVM also supports other foreign runtimes and virtual machines, anything that compiles to [WebAssembly (WASM)](https://developer.mozilla.org/en-US/docs/WebAssembly). WASM provides the flexibility we need to meet various needs because it was built for modern web browsers and supports multiple languages, including Rust and Go.
```